app/controllers/locators_controller.rb

```diff
@@ -0,0 +1,7 @@
+class LocatorsController < ApplicationController
+  def index
+    expires_in Rails.application.config.cache_max_age, public: true
+
+    render :index, layout: false
```

Comment:
Using the `layout false` class macro would mean we could drop this explicit `render` call here.